accession-commons-monotonic-generator-jpa/src/main/java/uk/ac/ebi/ampt2d/commons/accession/persistence/jpa/monotonic/service/ContiguousIdBlockService.java

```diff
@@ -91,7 +92,7 @@ public void save(ContiguousIdBlock block) {
         entityManager.flush();
     }
 
-    @Transactional(isolation = Isolation.SERIALIZABLE)
+    @Transactional(isolation = Isolation.SERIALIZABLE, propagation = Propagation.REQUIRES_NEW)
```

Comment:
I tried to read up on how this propagation works but I feel it would be good to get your explanation.